### ISSUE TITLE
[Help] Calling the randomizer with arguments

### ISSUE THREAD

**Issue by PanCave** [Ajarmar/universal-pokemon-randomizer-zx#708]
**Describe your problem clearly:** 
Is there a way to call the randomizer with arguments? Something like:
java -Xmx4608M -jar PokeRandoZX.jar -f my_rom.nds -s my_settings.rnqs -o randomized_rom.nds [-l randomized_rom.nds.log]

**Comment by Fellshadow** [comment]
yes that's possible, it's not documented i believe but the usage is similar to what you suggested:
 `java -Xmx4608M -jar PokeRandoZX.jar -i my_rom.nds -s my_settings.rnqs -o randomized_rom.nds [-l]` 
 
 essentially `-i` instead of `-f` for input file path, and `-l` (lowercase L) is just a boolean flag for whether to generate log files (it just uses the name of the output file for the log file, appending `.log`)

There's also `-d` and `-u` options for saving as a directory and update file path, respectively. These are options relevant to randomizing 3DS games i think.
